packages/carbon-web-components/src/components/accordion/accordion-item.ts

```diff
@@ -39,21 +39,7 @@ const observeResize = (observer: ResizeObserver, elem: Element) => {
   } as Handle;
 };
 
-/**
- * Accordion item.
- *
- * @element bx-accordion-item
- * @fires bx-accordion-item-beingtoggled
- *   The custom event fired before this accordion item is being toggled upon a user gesture.
- *   Cancellation of this event stops the user-initiated action of toggling this accordion item.
- * @fires bx-accordion-item-toggled - The custom event fired after this accordion item is toggled upon a user gesture.
- * @csspart expando The expando button.
- * @csspart expando-icon The expando icon.
- * @csspart title The title.
- * @csspart content The content.
- */
-@customElement(`${prefix}-accordion-item`)
-class BXAccordionItem extends FocusMixin(LitElement) {
+export class BXAccordionItemBase extends FocusMixin(LitElement) {
```

Comment:
Making all these base classes abstract might be a good idea, but it'll involve considerably more work because we'd need to also implement all abstract members on each subclass.

Thoughts @kennylam?

Reply:
> it'll involve considerably more work because we'd need to also implement all abstract members on each subclass

I wouldn't expect there to be any abstract members currently in any of these classes by definiton, otherwise they would have already needed to be declared abstract, so there shouldn't be any work in child classes.